server/src/main/kotlin/org/javacs/kt/KotlinProtocolExtensions.kt

```diff
@@ -9,4 +9,7 @@ import java.util.concurrent.CompletableFuture
 interface KotlinProtocolExtensions {
     @JsonRequest
     fun jarClassContents(textDocument: TextDocumentIdentifier): CompletableFuture<String?>
+
+    @JsonRequest
+    fun getBuildOutputLocation(): CompletableFuture<String?>
```

Comment:
Nit: I think dropping the `get` prefix here would be slightly more consistent with the naming of the other LSP methods